client/api/python/heketi/heketi.py

```diff
@@ -54,3 +56,535 @@ def hello(self):
         r = requests.get(self.host + uri, headers=headers)
         return r.status_code == requests.codes.ok
 
+
+class Cluster(object):
+    """ Class to run cluster operations """
+
+    def __init__(self, client):
+        self.client = client
+
+    def create(self):
+        headers = {}
+        method = 'POST'
+        uri = "/clusters"
+
+        output = ''
+        err = ''
+
+        self.client._set_token_in_header(headers, method, uri)
+        r = requests.post(self.client.host + uri, headers=headers)
+
+        if r.status_code == requests.codes.created:
+            output = r.json()
+        else:
+            err = r.status_code
+
+        return output, err
+
+
+
+    def info(self,cluster_id):
+        headers = {}
+        method = 'GET'
+        uri = "/clusters/" + cluster_id
+
+        output = ''
+        err = ''
+
+        self.client._set_token_in_header(headers, method, uri)
+        r = requests.get(self.client.host + uri, headers=headers)
+
+        if r.status_code == requests.codes.ok:
+            output = r.json()
+        else:
+            err = r.status_code
+
+        return output, err
+
+
+
+    def list(self):
+        headers = {}
+        method = 'GET'
+        uri = "/clusters"
+
+        output = ''
+        err = ''
+
+        self.client._set_token_in_header(headers, method, uri)
+        r = requests.get(self.client.host + uri, headers=headers)
+
+        if r.status_code == requests.codes.ok:
+            output = r.json()
+        else:
+            err = r.status_code
+
+        return output, err
+
+
+
+    def delete(self,cluster_id):
+        """ Delete cluster. Returns only \
+        error status code """
+
+        headers = {}
+        method = 'DELETE'
+        uri = "/clusters/" + cluster_id
+
+        err = ''
+
+        self.client._set_token_in_header(headers, method, uri)
+        r = requests.delete(self.client.host + uri, headers=headers)
+
+        if r.status_code != requests.codes.ok:
+            err = r.status_code
+
+        return err
+
+
+
+class Node(object):
+
+    def __init__(self, client):
+        self.client = client
+
+    def add(self, **kwargs):
+
+        addnode_params = kwargs
+        headers = {}
+        method = 'POST'
+        uri = "/nodes"
+        queue_loc = ''
+
+        output = ''
+        err = ''
+
+        self.client._set_token_in_header(headers, method, uri)
+        r = requests.post(self.client.host + uri,
+                          data = json.dumps(addnode_params),
+                          headers=headers)
+
+        if r.status_code == requests.codes.accepted:
+            queue_loc = r.headers['location']
+
+            if queue_loc:
```

Comment:
This section that pends should definitely be made a utility function of the _Client_ class so that other functions use it.